packages/components/tests/integration/components/hds/dropdown/index-test.js

```diff
@@ -119,4 +119,27 @@ module('Integration | Component | hds/dropdown/index', function (hooks) {
     assert.dom('#test-dropdown').hasAttribute('data-test1');
     assert.dom('#test-dropdown').hasAttribute('data-test2', 'test');
   });
+
+  // ACCESSIBILITY
+
+  test('it should render a list of items without a role if no selectable items are passed in', async function (assert) {
```

Comment:
question: I don't really understand this test, can you explain? Lists show up in the rotor and as such need an accessible name even if they don't have interactive elements.

Reply:
> it should render a list of items without a role if no selectable items are passed in

now that I read it, I can see how the double negation doesn't help

what I'm testing here is that _if_ we don't have selectable items (checkbox, radio), we don't add `role="listbox"` to the list, relying on the implicit role for an unordered list, which is "list" (in this scenario, a screen reader would read: _my item, button, group, 5 items_)

in the following test, we check that we add `role="listbox"` if there are selectable items (in this scenario, a screen reader would read: _my item, not selected, 1 of 5_)

I'm trying to make sure the `role` is only explicitly applied to the list if it has selectable items so that users are told about the state of each item when navigating through them